js/notifications.js

```diff
@@ -50,7 +50,7 @@ Fliplet.Widget.register('PushNotifications', function () {
       }
       else {
         Fliplet.Storage.set('fl_notification_update', data).then(function () {
-          Fliplet.Navigate.screen(data.page);
+          Fliplet.Navigate.screen(data.page, { transition: data.transition });
```

Comment:
@angelnikolov I think `Fliplet.Navigate.screen()` supports a few other properties for the second parameter too. Maybe pass them all through like this?

```suggestion
          Fliplet.Navigate.screen(data.page, data);
```

How is the query passed, anyway?

Reply:
Good call, will check.